crypto/bls12381/bls12381.go

```diff
@@ -178,11 +176,12 @@ func GenPrivKeyFromSecret(secret []byte) PrivKey {
 	return privKey.Serialize()
 }
 
+// ReverseBytes returns a reversed sequence bytes of passed slice
 func ReverseBytes(bz []byte) []byte {
```

Comment:
please remove redundant bls12381_test.reverseBytes() and use this instead.

Maybe move this to tmbytes?